src/frontend/src/sass/_default.scss

```
@@ -1,12 +1,61 @@
-:root {
-  box-sizing: border-box;
-}
+  @include breakpoint(0 map-get($grid-breakpoints, 'md')) {
+  :root {
+    box-sizing: border-box;
+    --custom-font-size: 14px;
+  }
+  }
+  @include breakpoint(map-get($grid-breakpoints, 'md')) {
+  :root {
+    box-sizing: border-box;
+    --custom-font-size: 16px;
+  }
+  }
```

will something like this work?
```suggestion
  :root {
    box-sizing: border-box;
    --custom-font-size: 16px;
    @include breakpoint(0 map-get($grid-breakpoints, 'md')) {
      --custom-font-size: 14px;
    }
  }  
   
```

---

oh.. i tried it. Doesn't work. I have to define it above root.